test/cbmc/proofs/OTA_HTTP_strerror/OTA_HTTP_strerror_harness.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * AWS IoT Over-the-air Update v3.0.0
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/**
+ * @file OTA_HTTP_strerror_harness.c
+ * @brief Implements the proof harness for OTA_HTTP_strerror function.
+ */
+
+/* http interface includes. */
+#include "ota_http_private.h"
+
+void OTA_HTTP_strerror_harness()
+{
+    OtaHttpStatus_t status;
+    const char * pErrorMsg = NULL;
+
+    pErrorMsg = OTA_HTTP_strerror( status );
+
+    /* The function returns a string which is never a NULL. If it does, then
+     *  there is a problem. */
+    __CPROVER_assert( pErrorMsg != NULL, "function cannot return null" );
```

Comment:
Is "function cannot return null" the error log that will be printed if this is triggered?

If so, please update this to be a complete sentence and to be more clear about what the failure was compared to what was expected.

Reply:
Updated it in 
6f40538